src/main/java/com/dev/cinema/model/dto/movie/MovieDtoMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.dev.cinema.model.dto.movie;
+
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieDtoMapper {
+    private final MovieService movieService;
```

Comment:
this class should only map dto to entity and otherwise but should not be responsible for adding entity to database. 
Let's use services in controllers. Same for other mappers

Reply:
done

Reply:
do we need this field here then?

Reply:
done